### ISSUE TITLE
Nodes getting stuck at Block Height 1

### ISSUE THREAD

**Issue by raghulraj** [MinaProtocol/mina#7497]
Node is getting stuck at Block Height:1 and Max observed length: 1

![image](https://user-images.githubusercontent.com/1759429/104660102-c4a3cc80-567a-11eb-9692-2c26efae95e7.png)
Logs are attached.
[2021-01-14_23-07-28.tar.gz](https://github.com/MinaProtocol/mina/files/5817462/2021-01-14_23-07-28.tar.gz)

**Comment by raghulraj** [comment]
```
Ledger Merkle root:                            jxX6VJ84HaafrKozFRA4qjnni4aPXqXC2H5vQLKSryNpKTXuz1R
Protocol state hash:                           3NLxYrjb7zmHdoFgBrubCN8ijM8v7eT8kvLiPLc9DHt3M8XrDDEG
Chain id:                                      2dbac725ebb5c6c8b1daf21cda9e3694951ab72afe9be99ff7ca47574112f43b
Git SHA-1:                                     [DIRTY]16ebdd6597ab050fdaf8a075cc39d9b7be1233cf
```

**Comment by averquik** [comment]
Also, I noticed about 10 more people with the same problem.
![image](https://user-images.githubusercontent.com/73314136/104660426-8a1c4d00-56d7-11eb-99b7-8fa4881395d2.png)

**Comment by mnguyen-io** [comment]
I have the same issue. On a previously running node (has been running and fine for 24 hours), I restarted the mina service and now it's stuck at block height 1/observed 1.

ii  mina-testnet-postake-medium-curves 0.2.2-1-b14e324                             amd64        Mina Client and Daemon

/usr/local/bin/coda daemon -peer-list-file /root/peers.txt -insecure-rest-server  -generate-genesis-proof true -log-level Info -file-log-level Info -super-catchup


```
Thu Jan 14 23:16:54 UTC 2021
{"timestamp":"2021-01-14 23:16:55.119336Z","level":"Debug","source":{"module":"Block_time","location":"File \"src/lib/block_time/block_time.ml\", line 76, characters 18-30"},"message":"Environment variable CODA_TIME_OFFSET not found, using default of 0","metadata":{"pid":20844}}
Coda daemon status
-----------------------------------

Global number of accounts:                     2026
Block height:                                  1
Max observed block length:                     1
Local uptime:                                  6m42s
Ledger Merkle root:                            jxX6VJ84HaafrKozFRA4qjnni4aPXqXC2H5vQLKSryNpKTXuz1R
Protocol state hash:                           3NLxYrjb7zmHdoFgBrubCN8ijM8v7eT8kvLiPLc9DHt3M8XrDDEG
Chain id:                                      2dbac725ebb5c6c8b1daf21cda9e3694951ab72afe9be99ff7ca47574112f43b
Git SHA-1:                                     [DIRTY]b14e32415b15af04865ca069af31cea7716be251
Configuration directory:                       /root/.coda-config
Peers (36):                                    
        /ip4/188.127.250.42/tcp/8302/p2p/12D3KooWBKR3xmcXf5gLEALko46kdLbZDs9jnGq7Dwx9oC6He19A
        /ip4/37.9.13.24/tcp/8302/p2p/12D3KooWJvu72HWg75664JXEPLrZT7H4P7KccX9qXeo6Endo4KGo
        /ip4/173.71.143.248/tcp/8302/p2p/12D3KooWDrVn96LC9WF9U4BRYG2g8YvsQet5bdFFzohzj3ZzLQQG
        /ip4/159.203.11.148/tcp/8302/p2p/12D3KooWRmXR8V8FxzxQu9fkyaMTqoP7QETH5t2UZBMV5sEFmWR5
        /ip4/35.226.95.52/tcp/10510/p2p/12D3KooWPXFnvusqHaKuuuUX5w54C6TKfXv7MQVrtnXDEsjS4Zcn
        /ip4/213.232.192.228/tcp/8302/p2p/12D3KooWH3y8JbMK54yCM7JQLKsksHvxcXPBskrohCa8yV138dnF
        /ip4/95.216.32.75/tcp/8302/p2p/12D3KooWHmxRoubY3HfXw3HhoMe8z9bQ5kyGVGQsCHwdvpWX7Xsf
        /ip4/34.66.157.200/tcp/10001/p2p/12D3KooWJ9mNdbUXUpUNeMnejRumKzmQF15YeWwAPAhTAWB6dhiv
        /ip4/35.188.176.148/tcp/10520/p2p/12D3KooWSkr77aJSMAWo7XnzcoMsfXEcq2ccsqiZkMWdyTSL3fWz
        /ip4/92.63.110.11/tcp/8302/p2p/12D3KooWMT2jSJKtKZmTUJJYKmUsfHVTZkpqeLAnaFbyuQtMUyzj
        /ip4/23.226.73.106/tcp/8302/p2p/12D3KooW9yCf9R6zqrpAWUqgyV3tnXFM7EwbYEd1WmXinTmjaWeW
        /ip4/146.148.67.24/tcp/8302/p2p/12D3KooWMHowmYFe39zdcUjaot3XPjjGWwA37KEp43kJrAmBNWVM
        /ip4/34.68.171.184/tcp/10518/p2p/12D3KooWQUjcrUAH5PnZKKTQ1ece6ZASgMJwzEXuUk9P4mytwYys
        /ip4/95.217.22.173/tcp/8302/p2p/12D3KooWQdynGruWxWkoEacmB8SdWo6oRFQVAkwJU8gg7u6PDhRF
        /ip4/116.203.243.65/tcp/8302/p2p/12D3KooWSPT3FfqZDNQXymNUbfnCZrkvWbKEKf9HJKC25RUQLZ2j
        /ip4/178.170.46.67/tcp/8302/p2p/12D3KooWDHfsARmB8Z95H2qWiPnfU387LAfx5yB3Wf6bduZe9CDS
        /ip4/161.97.121.167/tcp/8302/p2p/12D3KooWBZQepY3gCwdjufkWDn9qRLosRVb2xr7gHwDhkycXsD79
        /ip4/34.126.71.146/tcp/8302/p2p/12D3KooWKfPpiGnM4LMQhY6R7ehCLr3WHj3GB1em5ep9rJ1Sfrn6
        /ip4/88.198.106.180/tcp/8302/p2p/12D3KooWG7NJ4ym48xzR4PxGQrfHCVRVZYEJsSnEfx4rRQE6qtWr
        /ip4/35.193.175.21/tcp/10506/p2p/12D3KooWHHgxmedmJbeqhE3ee1FEEaVkrJG9yfAsjdAeDbdspVWm
        /ip4/164.68.106.218/tcp/8302/p2p/12D3KooWH6whDiPwAK7bcenWJdQ3KWC5LqmQjeqq8632HmUTBDyc
        /ip4/136.243.83.68/tcp/8302/p2p/12D3KooWPPACyFgxg5kbC4roUxEDDupDnedYyUPocpJS34VTuJZW
        /ip4/34.123.212.177/tcp/10507/p2p/12D3KooWKX4PKkC6NuVjVLvanutK3uc5BzAojCHwqW65oKksh6UN
        /ip4/34.68.198.209/tcp/10519/p2p/12D3KooWLRMHRQNLdXpmb2ZBFoPU6Vtt9vizGmEaR8qY6bbPnutW
        /ip4/188.34.153.55/tcp/8302/p2p/12D3KooWNvdPDv6AYNuVbbuWq9M1Gf6pD1np6MmUHhWrKSt8ViRj
        /ip4/47.56.184.255/tcp/8302/p2p/12D3KooWBYSTSRWviRJz3MhsfMEweKrXcG5V37pJyf4yJ9dXMwqF
        /ip4/195.201.195.221/tcp/8302/p2p/12D3KooWEHrafQnjaDLYYXfRXKYC7AgHhYPePMLQvFxUYSUmYSws
        /ip4/49.12.99.117/tcp/8302/p2p/12D3KooWRcBbpMEfkk4gyYk5Cny3wJfoVS96wHEXXML7HQ4NMW9Q
        /ip4/34.121.187.223/tcp/10517/p2p/12D3KooWDuVf77qd61T1332hqCvJWTDMkhrMuqoTiK9h29vjBgF9
        /ip4/35.193.91.26/tcp/10511/p2p/12D3KooWEFMJM3ru7q6pGWqei1RCcJTxdsHx6dGGXWntkxnJDkC2
        /ip4/34.71.83.44/tcp/10909/p2p/12D3KooWAFFq2yEQFFzhU5dt64AWqawRuomG9hL8rSmm5vxhAsgr
        /ip4/34.70.132.136/tcp/10508/p2p/12D3KooWFBNsTGhuH8pMcQhL5aZoFFjTedCZUtPSuevRPhvaZPXh
        /ip4/178.170.41.104/tcp/8302/p2p/12D3KooWBz78QM3Wvzzn8n6gau93jqmvMDtjr6qYahKcbrFc1eoG
        /ip4/135.181.158.223/tcp/8302/p2p/12D3KooWPq3jQJGjSSsTmX8bu8VkEQDy1EhveHudE3oYPPgdh2xC
        /ip4/142.93.64.60/tcp/8302/p2p/12D3KooWPFMw2RK25FXqC6gEAVMWXovtmv8hxwB1Cvo8nLMduQLR
        /ip4/95.216.38.228/tcp/8302/p2p/12D3KooWQ88Ajpa73ZqoPZSNXQaU7x4ihRzJVXiP885UfPwaAZ1E
User_commands sent:                            0
SNARK worker:                                  None
SNARK work fee:                                25000000
Sync status:                                   Synced
Catchup status:                                
        Finished:  1

Block producers running:                       0
Best tip consensus time:                       epoch=0, slot=0
Best tip global slot (across all hard-forks):  0
Next block will be produced in:                None this epoch… checking at in 14.582d
Consensus time now:                            epoch=1, slot=140
Consensus mechanism:                           proof_of_stake
Consensus configuration:                       
        Delta:                     0
        k:                         290
        Slots per epoch:           7140
        Slot duration:             3m
        Epoch duration:            14d21h
        Chain start timestamp:     2020-12-30 19:14:30.000000Z
        Acceptable network delay:  3m

```

**Comment by mnguyen-io** [comment]
On a **new node** on a separate cloud provider, I am seeing something similar.

ii  mina-testnet-postake-medium-curves 0.2.4-16ebdd6                               amd64        Mina Client and Daemon



```
root@mina-testworld-dev-2:~$ coda client status
{"timestamp":"2021-01-14 23:19:28.271772Z","level":"Debug","source":{"module":"Block_time","location":"File \"src/lib/block_time/block_time.ml\", line 76, characters 18-30"},"message":"Environment variable CODA_TIME_OFFSET not found, using default of 0","metadata":{"pid":13857}}
Coda daemon status
-----------------------------------

Global number of accounts:                     2026
Block height:                                  1
Max observed block length:                     1
Local uptime:                                  28m
Ledger Merkle root:                            jxX6VJ84HaafrKozFRA4qjnni4aPXqXC2H5vQLKSryNpKTXuz1R
Protocol state hash:                           3NLxYrjb7zmHdoFgBrubCN8ijM8v7eT8kvLiPLc9DHt3M8XrDDEG
Chain id:                                      2dbac725ebb5c6c8b1daf21cda9e3694951ab72afe9be99ff7ca47574112f43b
Git SHA-1:                                     [DIRTY]16ebdd6597ab050fdaf8a075cc39d9b7be1233cf
Configuration directory:                       /root/.coda-config
Peers:                                         50
User_commands sent:                            0
SNARK worker:                                  None
SNARK work fee:                                25000000
Sync status:                                   Synced
Catchup status:                                
        Finished:  1

Block producers running:                       0
Best tip consensus time:                       epoch=0, slot=0
Best tip global slot (across all hard-forks):  0
Next block will be produced in:                None this epoch… checking at in 14.58d
Consensus time now:                            epoch=1, slot=141
Consensus mechanism:                           proof_of_stake
Consensus configuration:                       
        Delta:                     0
        k:                         290
        Slots per epoch:           7140
        Slot duration:             3m
        Epoch duration:            14d21h
        Chain start timestamp:     2020-12-30 19:14:30.000000Z
        Acceptable network delay:  3m

Addresses and ports:                           
        External IP:    159.203.36.25
        Bind IP:        0.0.0.0
        Libp2p PeerID:  12D3KooWMZZaxXgQMHpy41cBwav3TbrGsJnpyDMFRyZCW1vx9dyj
        Libp2p port:    8302
        Client port:    8301
```

**Comment by SoDi1203** [comment]
[2021-01-14_23-08-56.tar.gz](https://github.com/MinaProtocol/mina/files/5817486/2021-01-14_23-08-56.tar.gz)

**Comment by raghulraj** [comment]
![image](https://user-images.githubusercontent.com/1759429/104674216-1d359280-5698-11eb-9333-bb1fb9bea8de.png)
Issue is fixed on Commit [DIRTY]d44a1339bdbdd5ebe747124c7146f435a3f73453

**Comment by SoDi1203** [comment]
thank you but how do you get 453 ?

[image: image.png]

**Comment by ghost-not-in-the-shell** [comment]
This has already fixed by #7494
